utils/netlink.go

```diff
@@ -41,6 +44,24 @@ func LinkContainerNS(nspath, containerName string) error {
 	return nil
 }
 
+// ContainerNSToPID resolves the name of a container via
+// the "/run/netns/<CONTAINERNAME>" to its PID
+func ContainerNSToPID(cID string) (int, error) {
+	pnns, err := filepath.EvalSymlinks("/run/netns/" + cID)
+	if err != nil {
+		return 0, err
+	}
+	pathElem := strings.Split(pnns, "/")
+	if len(pathElem) != 4 {
+		return 0, fmt.Errorf("unexpected result looking up container PID")
+	}
+	pid, err := strconv.Atoi(pathElem[1])
+	if err != nil {
+		return 0, fmt.Errorf("error converting the string part of the namespace link to int")
+	}
+	return pid, nil
+}
+
```

Comment:
let's move this to `utils/containers.go`

Reply:
is moved